### DISCUSSION TITLE
Show processes of all users

### DISCUSSION THREAD

**Discussion by sqcheseaux** [nicolargo/glances#2809] [Q&A]
Hi all

Is it possible to have `Glances` works the same way `Top` does: by default, showing the processes of all users without using `sudo`?

I use it on macOS and installed it using homebrew.

Best regards

**Comment by nicolargo** [comment]
It is not a Glances limitation, it is a system one.

By default Glances shows all processes. Nevertheless, if your system is secured it can hide processes that no belong to the current user.
